jaegertracing/jaegertracing.go

```diff
@@ -0,0 +1,193 @@
+/*
+Package tracing provides middleware to Opentracing using Jaeger.
+
+Example:
+```
+	package main
+	import (
+		"github.com/labstack/echo/v4"
+		"echo-tracing/middleware/jaegertracing"
```

Comment:
You probably need to fix this import and its use

Reply:
Done!